JASP-Desktop/widgets/boundqmltextinput.cpp

```diff
@@ -274,6 +274,19 @@ bool BoundQMLTextInput::isJsonValid(const Json::Value &optionValue)
 	return valid;
 }
 
+void BoundQMLTextInput::setUp()
+{
+	if (form())
+		// For unknown reason, when the language is changed, QML reset the default value.
```

Comment:
QML resets all bindings on language changed, this is not unknown, it is an implementation detail. The weird thing is that it only happens with these few elements...